padinfo/view_state/pic.py

```diff
@@ -10,34 +10,40 @@
 
 class PicViewState(ViewState):
     def __init__(self, original_author_id, menu_type, raw_query, query, color, monster: "MonsterModel",
+                 use_evo_scroll: bool = True,
```

Comment:
why optional

Reply:
in case anyone is reading this later - if these views are used outside of the `^id` command (which `IdViewState` is), it's possible we won't care about the `use_evo_scroll` preference and not have determined that so we don't require it